src/main/java/com/stripe/model/Card.java

```diff
@@ -34,6 +34,12 @@
 	String status;
 	String tokenizationMethod;
 
+	// Please note that these field are for internal use only and are not typically returned
+	// as part of standard API requests.
+	String description;
```

Comment:
+1. An explicit `private` would be better, but we these get `package-private` by default and at least won't leak out of the package.